fiboa_cli/datasets/fieldscapes_kenya_2022.py

```diff
@@ -0,0 +1,138 @@
+# TEMPLATE FOR A FIBOA CONVERTER
```

Comment:
I'd propose to rename the file to `fs_ke` for consistency.

Reply:
Not sure about this. Since int he meeting We (Hannah, Jacob, Me and Snehal) decided to keep a naming convention similar to boundaries_<country>_<year>.parquet to keep it a bit separate than the rest of the global dataset. We can discuss more on this I feel.

Reply:
This is just the converter name, not the file name?!

Reply:
Yes okay, I will cross check to make sure all the fieldscape PRs align.